ehsm_kms_service/router.js

```diff
@@ -197,6 +199,37 @@ const router = async (p) => {
       break
     case key_management_apis.DisableKey:
       disableKey(appid, payload, res, DB)
+    case remote_attestation_apis.GenerateQuote:
+      try {
+        const { challenge } = payload
+        if (challenge) {
+          napi_res = napi_result(action, res, [challenge])
+          napi_res && res.send(napi_res)
+        } else {
+          res.send(_result(400, 'Empty challenge', {}))
+        }
+      } catch (error) {}
+      break
+    case remote_attestation_apis.VerifyQuote:
+      try {
+        const { quote, nonce } = payload
+        if (quote && nonce) {
+          napi_res = napi_result(action, res, [quote, nonce])
+          if (napi_res) {
+            let sign = await gen_hmac(DB, appid, napi_res.result)
+            if (sign.length > 0) {
+              napi_res.result.sign = sign
+              res.send(napi_res)
+            } else {
+              res.send(_result(400, 'Generate sign error', {}))

```

Comment:
if there are any internal errors, we can log but not send the details to the caller. Or send a general "Internal error." 
If there are some errors caused by the caller like the wrong signature, wrong appid, unsupported API, we can send to the caller.
same @227.
@230 is OK because it's the error from the caller.